doc/examples/rmodel/distance.r

```diff
@@ -0,0 +1,28 @@
+args = commandArgs(trailingOnly=TRUE)
+pars = list()
+names_ = list()
+for (arg in args) {
+    s = strsplit(arg, "=")
+    names_ = c(names_, s[[1]][1])
+    pars = c(pars, s[[1]][2])
+}
+
+names(pars) = names_
+
+
+distance <- function(pars){
+    sumstat_0 = readRDS(pars$sumstat_0)
+    # insert observed data here directly
+    # note: by design, pyabc can currently not completely make sure
```

Comment:
does this mean this approach may not work as it should?

Reply:
currently, we always use the first argument for the simulated, and the second for the measured data. thus, it works at the moment. however, the structure of these arguments makes it not really clear. I think we should rename the arguments to ``simulated_sum_stat`` and ``measured_sum_stat`` at some point to ensure clarity.

btw: if the user mixes the order or sth changes, that will be immediately noticed by a huge error, so no problem.

Reply:
see #128